.github/workflows/test-pr.yml

```diff
@@ -112,11 +107,9 @@ jobs:
           container-name: kevm-ci-concrete-${{ github.sha }}
       - name: 'Build kevm-pyk'
         run: docker exec -u github-user kevm-ci-concrete-${GITHUB_SHA} /bin/bash -c 'make poetry'
-      - name: 'Build blockchain-k-plugin-deps'
-        run: docker exec -u github-user kevm-ci-concrete-${GITHUB_SHA} /bin/bash -c 'make CXX=clang++-14 plugin-deps'
-      - name: 'Build kevm'
-        run: docker exec -u github-user kevm-ci-concrete-${GITHUB_SHA} /bin/bash -c 'make build-kevm build-haskell build-llvm -j`nproc` RELEASE=true'
-      - name: 'Test kevm-pyk'
+      - name: 'Build targets'
+        run: docker exec -u github-user kevm-ci-concrete-${GITHUB_SHA} /bin/bash -c 'CXX=clang++-14 poetry -C kevm-pyk run kevm-dist --verbose build -j`nproc` plugin llvm haskell'
+      - name: 'Test integration'
```

Comment:
We have kinda inconsistent usage of `-j...` across the different tests. Can we just use ``-j`nproc``` everywhere?

Reply:
Done.